docs/integrations/ai-engines/rag.mdx

```diff
@@ -1,78 +1,73 @@
 ---
-title: Retrieval Augmented Generation
-sidebarTitle: Semantic Search (RAG)
+title: Semantic Search with RAG
+sidebarTitle: RAG
 ---
 
-This is a general-purpose Retrieval Augmented Generation handler that can be used to create, train, and depoy models within MindsDB.
+This documentation describes the Retrieval Augmented Generation integration that can be used to create, train, and deploy models within MindsDB.
 
 It supports the following:
 
 * Large language models such as [OpenAI](/integrations/ai-engines/openai) and [Writer](https://github.com/mindsdb/mindsdb/tree/staging/mindsdb/integrations/handlers/writer_handler#readme).
 * Vector databases such as [ChromaDB](/integrations/vector-db-integrations/chromadb) and FAISS.
```

Comment:
Let's add info that these vector databases are used to store sessions' data (not to provide training data).